Source/OxyPlot/OxyPlot.csproj

```diff
@@ -1,26 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <TargetFrameworks>netstandard1.0;netstandard2.0;net462</TargetFrameworks>
+        <TargetFrameworks>netstandard2.0;net462;net6.0</TargetFrameworks>
```

Comment:
Can we keep netstandard1.0?

Reply:
The motivation to remove it was that every platform which supports 1.0 but not 2.0 is long EOL, and the .NET Standard 1.0 is distributed as a granular set of packages so it makes package restores take a lot longer.

I don't particularly care.

Reply:
You've convinced me.